docs/sdks/uid2-sdk-ref-ios.md

```diff
@@ -0,0 +1,157 @@
+Install the iOS SDK via Swift Package Manager (SPM). There are two installation options:
+
+-   [Package.swift](#installing-with-packageswift)
+
+-   [Xcode](#installing-with-xcode)
+
+### Installing with Package.swift
+
+Add the following dependency to Package.swift:
+
+```
+dependencies: [
+.package(url: \"https://github.com/IABTechLab/uid2-ios-sdk.git\", from: "0.2.0"),
+]
+```
+
+### Installing with Xcode
+
+Add the following dependency via Xcode:
+
+![Xcode Dependency](images/xcode-uid2-ios-sdk.png)
+
+
+
+## How to Use the SDK
+
+The initial UID2 Identity *must* be generated by the implementing application and then passed into the UID2 SDK. Because of security requirements, the UID2 Mobile SDKs cannot create new UID2 identities.
+
+The UID2 Mobile SDKs can perform refreshes of UID2 identities, after an Identity is established. This is because the refresh functionality relies on the refresh tokens that are part of the UID2 Identity.
+
+The **UID2Manager** singleton is the primary developer API for the UID2 Android and iOS SDKs. It is responsible for storing, refreshing, and retrieving UID2 Identity.
+
+For iOS, the `UID2Manager` is initialized automatically the first time it is accessed. You can configure it to support automatic or manual refresh capabilities.
+
+## UID2Manager API
+
+This section includes the functions and variables that are part of the UID2Manager API.
+
+### Functions
+
+The following functions are available as part of the UID2Manager API:
+- [setIdentity()](#setidentity)
+- [resetIdentity()](#resetidentity)
+- [refreshIdentity()](#refreshidentity)
+- [getAdvertisingToken()](#getadvertisingtoken)
+- [setAutomaticRefreshEnabled()](#setautomaticrefreshenabled)
+
+#### setIdentity()
+
+Sets the UID2 Identity to be managed by the SDK.
+
+#### resetIdentity()
+
+Resets or removes the UID2 Identity currently being managed by the SDK.
+
+#### refreshIdentity()
+
+Manually refreshes the UID2 Identity being managed by the SDK.
+
+#### getAdvertisingToken()
+
+If the current UID2 Identity is valid, this function returns the UID2 token (advertising token).
+
+#### setAutomaticRefreshEnabled()
+
+Toggle for automatic refresh functionality.
+
+### Variables
+
+The following variables are available as part of the UID2Manager API:
+
+- [identity](#identity)
+- [identityStatus](#identitystatus)
+
+#### identity
+
+The Identity variable stores and returns the current UID2Identity data object being managed by the SDK.
+
+#### identityStatus
+
+The identityStatus variable stores and returns the status of the current UID2 Identity being managed by the SDK.
+
+## Code Samples
+
+The following code samples provide examples of performing specific activities relating to managing UID2 with the UID2 iOS SDK.
+
+Set the initial UID2 Identity:
+
+```swift
+UID2Manager.shared.setIdentity(_ identity: UID2Identity)
+```
+
+Get the UID2 token (advertising token) to pass to the Advertising SDK:
+
+```swift
+UID2Manager.shared.getAdvertisingToken()
+```
```

Comment:
Should we add a "For Use with Google Mobile Ads Integration" section (and ima later) here that would link back to the GMA page, something like:

_If you are using UID2 SDK for iOS to send UID2 Tokens to Google Mobile Ads (GAM), please read <GMA page>_

And add same thing for the android page

Reply:
Noted but commented out for now. iOS and Android will go out first, then GMA.